packages/runtime/src/helpers/verification.ts

```diff
@@ -105,6 +105,13 @@ export const checkForRootPublish = ({
 }
 
 export const checkZipSize = async (file: string, maxSize: number = LAMBDA_MAX_SIZE): Promise<void> => {
+  // Experimental: We can skip checking for the bundle size on the plugin side.
+  // Please reach out to the Netlify support team if you'd like to enable this feature
+  if (!process.env.NETLIFY_CHECK_BUNDLE_SIZE) {
```

Comment:
I'd thought this would be the other way around, where you'd set an environment variable if you want to bypass the size check. I don't see anyone going out of their way to enable this variable, so this would effectively amount to disabling the check?